elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -686,13 +686,13 @@ def add_kubernetes_pod_annotation(self, instance: KubernetesAnnotation, executio
         """Add KubernetesAnnotation instance to the execution object for the given runtime processor"""
         if "annotations" not in execution_object:
             execution_object["annotations"] = {}
-        execution_object["annotations"][instance.key] = instance.value
+        execution_object["annotations"][instance.key] = instance.value or ""
```

Comment:
Similar to what you're doing here for custom components, do we also need to ensure the value of labels and annotations, if not given, is an empty string (rather than `None`)?

if so, we can add something like this to lines 634 and 642 of the same file:

```
return {key: value or "" for key, value in labels.to_dict().items()}
```

Alternatively, we could also just coerce `None` to `""` during instance initialization, but I know that also comes with its own drawbacks.
```
return KubernetesLabel(key=key, value=value or "")
```

Reply:
> We can add something like this to lines 634 and 642 of the same file:

Done